app/src/test/java/org/mozilla/fenix/settings/wallpaper/ExtensionsTest.kt

```diff
@@ -54,6 +54,102 @@ class ExtensionsTest {
         assertEquals(downloadedSeasonalWallpapers, result[seasonalCollection])
     }
 
+    @Test
+    fun `GIVEN two collections of enough size WHEN grouped for onboarding THEN result contains 3 seasonal and 2 classic`() {
+        val seasonalCollection = getSeasonalCollection("finally fall")
+        val seasonalWallpapers = (0..5).map { generateSeasonalWallpaperCollection("${seasonalCollection.name}$it", seasonalCollection.name) }
+        val classicFirefoxWallpapers = (0..5).map { generateClassicFirefoxWallpaper("firefox$it") }
+        val allWallpapers = listOf(Wallpaper.Default) + classicFirefoxWallpapers + seasonalWallpapers
+
+        val result = allWallpapers.groupForWallpaperOnboarding()
+
+        assertEquals(3, result.count { it.collection.name == "finally fall" })
+        assertEquals(2, result.count { it.collection.name == classicCollection.name })
+        assert(result.contains(Wallpaper.Default))
```

Comment:
Instead of using `assert`, we generally do `assertTrue` or `assertFalse` to be more explicit.

Reply:
IIRC `assert` won't actually trigger test failures in some cases either, as it's a keyword rather than part of the JUnit framework